Client/Frontend/Browser/BrowserViewController/BrowserViewController+ProductNotification.swift

```diff
@@ -0,0 +1,323 @@
+// Copyright 2020 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import BraveShared
+import BraveUI
+import Shared
+
+// MARK: - ProductNotification
+
+extension BrowserViewController {
+    
+    // MARK: BenchmarkTrackerCountTier
+    
+    enum BenchmarkTrackerCountTier: Int, Equatable, CaseIterable {
+        case specialTier = 1000
+        case newbieExclusive = 5000
+        case casualExclusive = 10000
+        case regularExclusive = 25000
+        case expertExclusive = 75000
+        case professionalTier = 100000
+        case primeTier = 250000
+        case grandTier = 500000
+        case legendaryTier = 1000000
+        
+        var title: String {
+            switch self {
+                case .specialTier:
+                    return "Congratulations. You’re pretty special."
+                case .newbieExclusive, .casualExclusive, .regularExclusive, .expertExclusive:
+                    return "Congratulations. You’re part of an exclusive club."
+                case .professionalTier:
+                    return "Congratulations. You joined the pros."
+                case .primeTier:
+                    return "Congratulations. You’ve become a master."
+                case .grandTier:
+                    return "Congratulations. You’ve become a Grand Master."
+                case .legendaryTier:
+                    return "Congratulations. You are legendary."
+            }
+        }
+        
+        var nextTier: BenchmarkTrackerCountTier? {
```

Comment:
You don't have to write out every case like that :) You can do:

```swift
var nextTier: BenchmarkTrackerCountTier? {
    guard let indexOfSelf = Self.allCases.index(where: { self == $0 }) else {
        return nil
    }
    return Self.allCases[safe: indexOfSelf + 1]
}
```
    
So that it automatically indexes to the next case because your enum is `CaseIterable` and the above will use the `allCases` array.

Reply:
Didnt wanna make calculation but yeah, there might be even more cases here so, will swap :)